src/BlockTypes/ProductQuery.php

```diff
@@ -82,18 +82,19 @@ protected function initialize() {
 	}
 
 	/**
-	 * Extra data passed through from server to client for block.
-	 *
-	 * @param array $attributes  Any attributes that currently are available from the block.
-	 *                           Note, this will be empty in the editor context when the block is
-	 *                           not in the post content on editor load.
-	 */
-	protected function enqueue_data( array $attributes = [] ) {
-		parent::enqueue_data( $attributes );
-
-		$gutenberg_version = '';
+	 * Post Template support for grid view was introduced in Gutenberg 16 / WordPress 6.3
+	 * Fixed in:
+	 * - https://github.com/woocommerce/woocommerce-blocks/pull/9916
+	 * - https://github.com/woocommerce/woocommerce-blocks/pull/10360
+	 */
+	private function check_if_post_template_has_support_for_grid_view() {
+		if ( version_compare( $GLOBALS['wp_version'], '6.2.2', '>' ) ) {
```

Comment:
Are you sure that this check is correct? Should be 
```suggestion
		if ( version_compare( $GLOBALS['wp_version'], '6.3.0', '>=' ) ) {
```

Reply:
`version_compare` consider versions with suffix as lower, for example, 6.3.0 is higher than 6.3.0-RC2. That means the fix won't be applied to the RC version.

But you're right, I cannot compare to 6.2.2, because there may be another 6.2.x release and there will be incorrect behaviour.

I'll try to use this custom function [`wp_version_compare`](https://github.com/woocommerce/woocommerce-blocks/blob/b4ba06a6242cbb6a64d2ec554a263ebe60d8d3af/src/Utils/Utils.php#L16) which I belive was written for exactly this case or I'll think of something custom.

Reply:
> version_compare consider versions with suffix as lower, for example, 6.3.0 is higher than 6.3.0-RC2. That means the fix won't be applied to the RC version.

TIL

> version_compare consider versions with suffix as lower, for example, 6.3.0 is higher than 6.3.0-RC2. That means the fix won't be applied to the RC version.

Thanks!

Reply:
OK, code updated! And I updated testing steps to cover cases with higher and lower than WP 6.3 + Gutenberg 16 enabled and disabled.